### PR TITLE
Removed deprecated function

### PR DESCRIPTION
**Pull request by Miro-Collas**
Removed deprecated function. Ref https://github.com/dokufreaks/plugin-comment/issues/16

### CODE REVIEW

File: syntax.php [hunk 1]
```diff
@@ -10,7 +10,6 @@
 if(!defined('DOKU_INC')) die();
 
 if(!defined('DOKU_PLUGIN')) define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
-require_once(DOKU_PLUGIN.'syntax.php');
```
Comment (Klap-in) [inline syntax.php:13]: The two lines above this line and the comment can be deleted as well.

Comment (Miro-Collas) [reply syntax.php:13]: Aha ok thanks. :-)  I was just looking at what lines of code were throwing warnings within Dokuwiki.

**Comment by Klap-in** [comment]
Thanks for contributing! I have deleted the other lines as well.

**Comment by Miro-Collas** [comment]
> Thanks for contributing! I have deleted the other lines as well.

Great, thanks so much for the plugin! :-)

**Pull request merged**